controllers/httproute_controller.go

```diff
@@ -378,3 +408,40 @@ func isExternalDNSSupported(mgr ctrl.Manager) (bool, error) {
 	}
 	return false, nil
 }
+
+func (r *HTTPRouteReconciler) getBackendRefsIpFamiliesError(ctx context.Context, httpRoute *gateway_api.HTTPRoute) error {
```

Comment:
getXxxError kind of weird name, yes it returns error, but function name should tell meaning not implementation.
Just looking at code I would assume it does validation for IP families, that can be a name